widgets/HTTP-batchsource.json

```diff
@@ -0,0 +1,529 @@
+{
+  "metadata": {
+    "spec-version": "1.5"
+  },
+  "configuration-groups": [
+    {
+      "label": "Reference",
+      "properties": [
+        {
+          "widget-type": "textbox",
+          "label": "Reference Name",
```

Comment:
let's move this into General instead of having it's own section.

Reply:
done